test/server/cards/05_LOF/units/JediInHiding.spec.ts

```diff
@@ -0,0 +1,91 @@
+describe('Jedi In Hiding', () => {
+    integration(function (contextRef) {
+        it('Jedi In Hiding\'s when defeated ability may allow the player to use the force. If they do, opponent discards a card from their hand', async () => {
+            await contextRef.setupTestAsync({
+                phase: 'action',
+                player1: {
+                    groundArena: ['jedi-in-hiding'],
+                    hasForceToken: true
+                },
+                player2: {
+                    hand: ['no-glory-only-results', 'takedown'],
+                    hasInitiative: true,
+                }
+            });
+
+            const { context } = contextRef;
+
+            context.player2.clickCard(context.takedown);
+            context.player2.clickCard(context.jediInHiding);
+
+            expect(context.player1).toHavePassAbilityPrompt('Use the Force to make your opponent discards a card from their hand');
+            context.player1.clickPrompt('Trigger');
+
+            expect(context.player2).toBeAbleToSelectExactly([context.noGloryOnlyResults]);
+            expect(context.player2).not.toHaveChooseNothingButton();
+            expect(context.player2).not.toHavePassAbilityButton();
+
+            context.player2.clickCard(context.noGloryOnlyResults);
+
+            expect(context.player1).toBeActivePlayer();
+            expect(context.player1.hasTheForce).toBeFalse();
+            expect(context.noGloryOnlyResults).toBeInZone('discard', context.player2);
+        });
+
+        it('Jedi In Hiding\'s when defeated ability may allow the player to use the force. If they do, opponent discards a card from their hand', async () => {
```

Comment:
I think this test case name needs to be updated to clarify that this case is verifying that the effect cannot be used if the player does not have the force